ext/node/polyfills/internal_binding/udp_wrap.ts

```
@@ -300,7 +316,8 @@ export class UDP extends HandleWrap {
   }
 
   setBroadcast(_bool: 0 | 1): number {
-    notImplemented("udp.UDP.prototype.setBroadcast");
+    this.#listener?.setBroadcast(_bool === 1);
+    return 0;
```

This should return an error code when `setBroadcast` fails